adafruit_minimqtt/adafruit_minimqtt.py

```diff
@@ -114,7 +115,6 @@ class MQTT:
     :param bool is_ssl: Sets a secure or insecure connection with the broker.
     :param bool log: Attaches a logger to the MQTT client, defaults to logging level INFO.
     :param int keep_alive: KeepAlive interval between the broker and the MiniMQTT client.
-
```

Comment:
This blank line should remain here
> Insert a blank line after all docstrings (one-line or multi-line) that document a class
 https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
Personally I'm not a fan of the black linter but request granted.